docs/clients_tfs.md

```diff
@@ -167,24 +213,43 @@ When creating a Python-based client application, there are two packages on PyPi
         predict_response = prediction_service_stub.Predict(predict_request, 10.0)
         results = make_ndarray(predict_response.outputs["output_name"])
 
+.. tab:: curl
+
+    .. code-block:: sh  
+
+        curl http://localhost:9000/v1/models/my_model
+
 @endsphinxdirective
 
 ### Request Prediction on a Numpy Array
 
 @sphinxdirective
 
-.. tab:: ovmsclient
+.. tab:: ovmsclient [GRPC]
 
     .. code-block:: python
 
         import numpy as np
         from ovmsclient import make_grpc_client
 
-        client = make_grpc_client("10.20.30.40:9000")
+        client = make_grpc_client("localhost:9000")
         data = np.array([1.0, 2.0, ..., 1000.0])
         inputs = {"input_name": data}
         results = client.predict(inputs=inputs, model_name="my_model")
 
+.. tab:: ovmsclient [REST]
+
+    .. code-block:: python
+
+        import numpy as np
+        from ovmsclient import make_http_client
+
+        client = make_http_client("localhost:9000")
+        model_metadata = client.get_model_metadata(model_name="my_model", model_version="", 10.0)
+
+        data = np.array([1.0, 2.0, ..., 1000.0])
+        inputs = {"input_name": data}
+        results = client.predict(inputs=inputs, model_name="my_model", model_version="", 10.0)
```

Comment:
Same as for status.
port could be different and additional parameters are not relevant here